cmd/miner/main.go

```diff
@@ -10,12 +10,13 @@ import (
 	"os"
 	"os/signal"
 	"runtime"
+	"syscall"
 )
 
 func main() {
 	// Listen for interrupt signals.
 	interrupt := make(chan os.Signal, 1)
-	signal.Notify(interrupt, os.Interrupt)
```

Comment:
I'm pretty sure this won't build on windows.  It needs to be a separate file that only builds on unix.

Reply:
Yep I'm pretty sure you are correct, however, as @chappjc [commented on dcrseeder](https://github.com/decred/dcrseeder/pull/41#discussion_r553845018) some time ago, I don't expect anybody would run dcrpool on Windows.

We can move this into a separate file so it only builds on specific systems, as is done in dcrd and dcrwallet. What do you think @dnldd?

Reply:
Yeah that seems to be the way to go.

Reply:
Updated